provider/cluster/manager.go

```diff
@@ -155,10 +154,11 @@ func (dm *deploymentManager) run() {
 		dm.log.Debug("hostnames released")
 	}()
 
+	dm.startWithdrawal()
```

Comment:
It's hard to see why this moved. It looks like this starts immediately now. Can we put a comment explaining why this has to be started here?